pages/home.vue

```diff
@@ -0,0 +1,296 @@
+
+<template>
+  <div id="page">
+    <b-container style="background-color: white">
+      <b-row>
+        <b-col>
+          <div id="drop">
+            <b-select v-model="selectedValidated" :options="optionsValidated" @change="updateFilters" />
```

Comment:
It can be possible to create a custom components `<b-select>` , with props corresponding to the `v-model` and `options`